LWM2M 1018 Whats New entries (#323)
SoftwareAG/c8y-releasenotes: content/release-10-18-0/whatsnew-10-18-0.md

#### Authentication with OAuth2 access tokens from authorization servers

On top of standard SSO, Cumulocity IoT now also allows you to access the platform resources using access tokens from third-party authorization server directly as a Bearer token. The SSO configuration has been extended accordingly. For details refer to [Configuring single sign-on > Configuring authentication with OAuth2 access tokens from authorization servers](https://cumulocity.com/guides/10.18.0/users-guide/administration/#configuring-authentication-with-oauth2-access-tokens-from-authorization-servers) in the *User guide*.

### Device Management

#### LWM2M external decoder event execution improvements

> LWM2M Agent has parallelization improvements for the external decoder event executions which results in faster forward of requests to the external decoder microservices for the LWM2M object resources that has this custom action functionality selected in [LWM2M device protocol](https://www.cumulocity.com/guides/10.18.0/protocol-integration/lwm2m/#device-protocols).

Reviewer: or even "... less decoding delay"

#### Device deletion improvement for the LWM2M devices

> Starting from version 10.18.0, a LWM2M device can be completely deleted from the [All devices](https://cumulocity.com/guides/10.18.0/users-guide/device-management/#viewing-devices) list without the need of the additional execution of the LWM2M Connector device operation called `cleanupClusterTenantObjectsForEndpoint`, which was required before to be executed in order to re-register the same device. 

Reviewer: Sentence is too long.

> Starting from version 10.18.0, a LWM2M device can be completely deleted from the [All devices](https://cumulocity.com/guides/10.18.0/users-guide/device-management/#viewing-devices) list without the need of the additional execution of the LWM2M Connector device operation called `cleanupClusterTenantObjectsForEndpoint`, which was required before to be executed in order to re-register the same device. 

Reviewer 2: is "LWM2M Connector" the name? (asking because of the capital C)

Author: thanks. I'll change it to lower case c

#### Authentication with OAuth2 access tokens from authorization servers

On top of standard SSO, Cumulocity IoT now also allows you to access the platform resources using access tokens from third-party authorization server directly as a Bearer token. The SSO configuration has been extended accordingly. For details refer to [Configuring single sign-on > Configuring authentication with OAuth2 access tokens from authorization servers](https://cumulocity.com/guides/10.18.0/users-guide/administration/#configuring-authentication-with-oauth2-access-tokens-from-authorization-servers) in the *User guide*.

### Device Management

#### LWM2M external decoder event execution improvements

> The forwarding of device data to LWM2M decoder microservices (LWM2M device protocol [custom actions](https://cumulocity.com/guides/10.18.0/protocol-integration/lwm2m/#custom-actions)) has been significantly improved. By using a different internal mechanism to parallelize decoder invocations, the received data is now passed to the microservices at a faster rate, leading less decoding delay.

Reviewer 2: ```suggestion
The forwarding of device data to LWM2M decoder microservices (LWM2M device protocol [custom actions](https://cumulocity.com/guides/10.18.0/protocol-integration/lwm2m/#custom-actions)) has been significantly improved. By using a different internal mechanism to parallelize decoder invocations, the received data is now passed to the microservices at a faster rate, which reduces the decoding delay.
```

#### Device deletion improvement for the LWM2M devices

> Starting from version 10.18.0, a LWM2M device can be completely deleted from the [All devices](https://cumulocity.com/guides/10.18.0/users-guide/device-management/#viewing-devices) list. The additional execution of the LWM2M connector device operation called `cleanupClusterTenantObjectsForEndpoint` is not anymore required in order to re-register the same device.

Reviewer 2: ```suggestion
Starting from version 10.18.0, a LWM2M device can be completely deleted from the [All devices](https://cumulocity.com/guides/10.18.0/users-guide/device-management/#viewing-devices) list. The additional execution of the LWM2M connector device operation called `cleanupClusterTenantObjectsForEndpoint` is no longer required in order to re-register the same device.
```
